src/tests/ie_test_utils/functional_test_utils/layer_tests_summary/summarize.py

```diff
@@ -110,8 +110,11 @@ def collect_statistic(root: ET.Element, is_conformance_mode: bool):
         trusted_ops[device.tag] = 0
         covered_ops[device.tag] = 0
         for op in results[device.tag]:
+            # for correct display of reports without hanged item in report.xml
```

Comment:
minor: Please check the changes using CI

Reply:
i will add it to jira